ci/test/00_setup_env_native_tsan.sh

```diff
@@ -7,7 +7,7 @@
 export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_native_tsan
-export PACKAGES="clang-8 llvm-8 python3-zmq qtbase5-dev qttools5-dev-tools libevent-dev bsdmainutils libboost-filesystem-dev libboost-test-dev libboost-thread-dev libdb5.3++-dev libminiupnpc-dev libzmq3-dev libqrencode-dev"
+export PACKAGES="clang-15 llvm-15 python3-zmq qtbase5-dev qttools5-dev-tools libevent-dev bsdmainutils libboost-filesystem-dev libboost-test-dev libboost-thread-dev libdb5.3++-dev libminiupnpc-dev libzmq3-dev libqrencode-dev"
```

Comment:
19008: clang was bumped to 9, not 15

Reply:
I did it to unify with actual clang that we do use.
See this line in the same file below: https://github.com/dashpay/dash/pull/5722/commits/7f3a072e1e3dcb2601222c2cb1cb22e8b3ffc10a#diff-c78508382898de3637763e5fa2dd922ef957607779effe4a97c005aa6e3c2b77R15

This changes should be done there: #5375 but they weren't. 
On other hand Bitcoin jumped from 13 to 16th version of clang so there's no more suitable backport than this one.

Reply:
should maybe also backport the one that bumps it in `00_setup_env_native_ubsan.sh` too